analyzer/testdata/src/debug/negative_positive_replacements_test.go

```diff
@@ -0,0 +1,56 @@
+package debug
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNegativeReplacements(t *testing.T) {
```

Comment:
your replacement 

```go
{Fn: "Greater", Argsf: "0, a", ReportMsgf: report, ProposedFn: "Positive", ProposedArgsf: "a"},
```

is invalid because of arguments order

```go
// Greater asserts that the first element is greater than the second
//
//	assert.Greater(t, 2, 1)
```

But it is difficult to explore, because for `a == 0` both options (`Greater(a,0)` and `Positive(a)`) return `false`.

I fixed it and wrote replacement test to debug and be sure that checker suggestions is right.

Reply:
Oh, thanks for noticing it.